db/migrate/20220818171849_move_tl_user_history_to_previous_and_new_value.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class MoveTlUserHistoryToPreviousAndNewValue < ActiveRecord::Migration[7.0]
+  def change
+    execute <<~SQL
+      UPDATE user_histories
+      SET previous_value = old_tl,
+          new_value = new_tl,
+          details = NULL
+      FROM (
+          SELECT id user_history_id,
```

Comment:
Have you tried the migration on bad values?

Reply:
Yes. If the REGEXP_MATCHES does not return anything, then there will be no row for that user history and it will not be updated.